geopandas/base.py

```diff
@@ -91,7 +91,10 @@ def _invalidate_sindex(self):
 
         """
         self._sindex = None
-        self._sindex_valid = False
+
+    @property
+    def sindex_valid(self):
```

Comment:
Is this needed as a public property? (Is there a usecase beyond checking `sindex` as being None?)